access/access/README.md

```diff
@@ -0,0 +1,133 @@
+# Access Control Module
+
+This module provides authentication and authorization services for CADC (Canadian Astronomy Data Centre) applications.
+
+## Overview
+
+The access module is responsible for:
+- User authentication via SSO (Single Sign-On)
+- Cookie-based session management
+- Password reset functionality
+- User information services
+
+## Build Requirements
+
+- Java 11 or higher
+- Gradle 6.8.3 or higher
+
+## Runtime Configuration
+
+### Required Properties Files
+
+The following properties files need to be configured for runtime:
+
+#### 1. AccessControl.properties
+Located at: `src/main/resources/AccessControl.properties`
+
+```properties
+# SSO server configuration
+SSO_SERVERS = jenkinsd.cadc.dao.nrc.ca www.canfar.net www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca
+
+# Cookie domain configuration
+COOKIE_DOMAINS = www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca www.canfar.net
+
+# SSO token lifetime (default: 47 hours)
+SSO_TOKEN_LIFETIME_SECONDS = 169344
+```
+
+#### 2. RSA Key Files
+The module requires RSA key pairs for signing SSO cookies:
+
+- **Private Key**: `RsaSignaturePriv.key` (PEM format)
+- **Public Key**: `RsaSignaturePub.key` (PEM format)
+
+**Default locations**:
+- Production: Classpath root or system property `cadc.rsa.privkey.path`
+- Test: `src/test/resources/RsaSignaturePriv.key`
+
+### Key Generation
+
+For testing environments, keys are automatically generated using:
+```bash
+java -cp "src/test/java:build/classes/java/main" ca.nrc.cadc.accesscontrol.GenerateTestKey
+```
+
+For production, generate keys using:
+```bash
+# Generate 2048-bit RSA key pair
+openssl genrsa -out RsaSignaturePriv.key 2048
+openssl rsa -in RsaSignaturePriv.key -pubout -out RsaSignaturePub.key
+```
+
+## Deployment
+
```

Comment:
the baldur/README has the standard recommended commands to build, check, and run; that normally means the current dir is the module and not the repo base dir. 

although we include gradle wrapper in the repo, devs are free to use it or gradle (installed in some other way) as they see fit, or gradle within an IDE